lib/redis/store/namespace.rb

```diff
@@ -23,8 +23,12 @@ def get(key, options = nil)
         namespace(key) { |k| super(k, options) }
       end
 
-      def exists(key)
-        namespace(key) { |k| super(k) }
+      def exists(*keys)
+        super(*keys.map { |key| interpolate(key) })
+      end
+
+      def exists?(*keys)
+        super(*keys.map { |key| interpolate(key) })
```

Comment:
why not just call `exists(*keys)` here?

Reply:
Because they are two different methods, `exists?` being basically `exist() > 0`

Reply:
oh i see, they have different return types...got it.